docs/make.jl

```diff
@@ -52,17 +52,17 @@ end
 #####
 
 example_pages = [
-    "One-dimensional diffusion"          => "generated/one_dimensional_diffusion.md",
-    "Two-dimensional turbulence"         => "generated/two_dimensional_turbulence.md",
-    "Internal wave"                      => "generated/internal_wave.md",
-    "Convecting plankton"                => "generated/convecting_plankton.md",
-    "Ocean wind mixing and convection"   => "generated/ocean_wind_mixing_and_convection.md",
-    "Langmuir turbulence"                => "generated/langmuir_turbulence.md",
-    "Baroclinic adjustment"              => "generated/baroclinic_adjustment.md",
-    "Kelvin-Helmholtz instability"       => "generated/kelvin_helmholtz_instability.md",
-    "Shallow water Bickley jet"          => "generated/shallow_water_Bickley_jet.md",
-    "Horizontal convection"              => "generated/horizontal_convection.md",
-    "Tilted bottom boundary layer"       => "generated/tilted_bottom_boundary_layer.md"
+    # "One-dimensional diffusion"          => "generated/one_dimensional_diffusion.md",
+    # "Two-dimensional turbulence"         => "generated/two_dimensional_turbulence.md",
+    # "Internal wave"                      => "generated/internal_wave.md",
```

Comment:
Are these comments here on purpose for testing?

Reply:
Definitely my bad :)